node/harmony/node_explorer.go

```diff
@@ -43,7 +43,15 @@ func (node *Node) explorerMessageHandler(ctx context.Context, msg *msg_pb.Messag
 				Msg("[Explorer] onCommitted unable to parse msg")
 			return err
 		}
-
+		// If a future block is received, it logs a warning and discards the block
+		// TODO: can be done with isRightBlockNumAndViewID(recvMsg)
+		if recvMsg.BlockNum != node.Blockchain().CurrentBlock().NumberU64()+1 {
+			utils.Logger().Warn().
+				Uint64("Received BlockNum", recvMsg.BlockNum).
+				Uint64("Consensus BlockNum", node.GetConsensusBlockNum()).
```

Comment:
Explorer node does not participate in consensus, so getting consensus block number doesn't make much sense for me

Reply:
Fixed